pandas/tests/extension/test_arrow.py

```diff
@@ -1501,8 +1511,8 @@ def test_to_numpy_int_with_na():
     data = [1, None]
     arr = pd.array(data, dtype="int64[pyarrow]")
     result = arr.to_numpy()
-    expected = np.array([1, pd.NA], dtype=object)
-    assert isinstance(result[0], int)
+    expected = np.array([1, np.nan])
```

Comment:
Does this mean that `ArrowExentionArray.to_numpy` is now lossy for integer types if there are NA's since it will now be converted to float?

Reply:
Yes

Reply:
I'm a little surprised it doesn't break more tests as I think there are still a number of places that try to round-trip through numpy

Reply:
yeah same